lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -2353,6 +2353,10 @@ StateType ProcessGDBRemote::SetThreadStopInfo(StringExtractor &stop_packet) {
         uint32_t reg = UINT32_MAX;
         if (!key.getAsInteger(16, reg))
           expedited_register_map[reg] = std::string(std::move(value));
+      } else if (key.compare("swbreak") == 0 || key.compare("hwbreak") == 0) {
+        // swbreak and hwbreak are also expected keys, but we don't need to
+        // change our behaviour for them because lldb always expects the remote
+        // to adjust the program counter (if relevant, e.g., for x86 targets)
```

Comment:
I don't think we need the actual `else if`. Since `if condition do nothing` looks like a programming mistake. The comment is fine on its own here.

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/102873/commits/d61ea562c82eab27ae2cc460b37f2f0d27961c1c